src/python/gen-rpk-help-doc.py

```diff
@@ -0,0 +1,231 @@
+import subprocess
+import re
+
+class Flag:
+  def __init__(self, value, type,explanation):
+    self.value = value
+    self.type = type
+    self.explanation = explanation
+
+def execute_process(commands):
+    if(len(commands)>0):
+        commands = commands[0].split(" ")
+    commands.insert(0,"rpk")
+    commands.append("-h")
+    process = subprocess.run(commands, stdout=subprocess.PIPE)
+    return process.stdout.decode("utf-8")
+
+def get_explanation(process_line):
+    return process_line[:process_line.find("Usage")].rstrip("\n")
+
+def get_usage(process_line):
```

Comment:
For each function which receives a line and parses it, could you please add comments about the structure of the line it expects, and the output?